src/compiler/scala/tools/nsc/transform/Delambdafy.scala

```diff
@@ -34,6 +34,7 @@ import scala.collection._
 abstract class Delambdafy extends Transform with TypingTransformers with ast.TreeDSL with TypeAdaptingTransformer {
   import global._
   import definitions._
+  import postErasure.{elimErasedValueType => postErase}
```

Comment:
maybe more confusing than helping? i'd just use `postErasure.elimErasedValueType` at the callsites, some verbosity doesn't hurt for cross-phase-calls.